### PR TITLE
Fix `TRACE` log level support

### PR DESCRIPTION
**Pull request by igor-sirotin**
Requires https://github.com/status-im/status-desktop/pull/12916
Requires https://github.com/status-im/status-desktop/pull/12934

### What does the PR do

1. Pass correct value to status-go log level when `TRACE` logs are used.
`TRACE` is not supported by status-go, so we replace it with `DEBUG`.

2. Fix places that were checking for `LOG_LEVEL` env variable.
- Use `STATUS_RUNTIME_LOG_LEVEL` instead
- Aldo check if `--log-level` CLI option is set

### CODE REVIEW

**Review by saledjenic — COMMENTED** [review]
@igor-sirotin feel free to discard/close this PR as already completed.

**Comment by igor-sirotin** [comment]
> @igor-sirotin the issue is already fixed in this PR #12934

I don't think it is. This is what I get for `TRACE` logs i your branch:
image

We pass `TRACE` to status-go, but zap logger doesn't have such log level.

**Comment by saledjenic** [comment]
> We pass TRACE to status-go, but zap logger doesn't have such log level.

@igor-sirotin  ahh I see now this post

Ok, so aren't the changes you did in `src/app_service/service/accounts/service.nim` enough for that?

I mean, we just need to pass debug to statusgo, wherever we see trace, isn't that enough?

File: src/app_service/service/node_configuration/service.nim [hunk 1]
```diff
@@ -273,6 +273,7 @@ proc isDebugEnabled*(self: Service): bool =
   return self.getLogLevel() == $LogLevel.DEBUG
 
 proc setLogLevel*(self: Service, logLevel: LogLevel): bool =
+  echo "<<< node_configuration.service.setLogLevel " & $logLevel
```
Comment (saledjenic) [inline src/app_service/service/node_configuration/service.nim:276]: leftover

**Review by saledjenic — APPROVED** [review]

**Pull request merged**